ergoCubSN000/sensors/RealSense_conf.ini

```diff
@@ -0,0 +1,13 @@
+device       RGBDSensorWrapper
+subdevice    realsense2withIMU
+name         /depthCamera
+
+[SETTINGS]
+depthResolution (640 480)    #Note the parentesys
+rgbResolution   (640 480)
+framerate       30
+enableEmitter   true
+alignmentFrame  RGB
+
+[HW_DESCRIPTION]
+clipPlanes (0.2 10.0)
```

Comment:
Hi @mfussi66 @Nicogene 

Would you mind checking whether we need this file too?
Just asking @mfussi66 as he recently proposed a PR on RealSense (see #430).

Reply:
This file is needed for running the device via `yarpdev`, if we want to enforce the usage of `yarprobotinterface` we have to remove it

Reply:
I'd suggest removing it since the RGBDSensorWrapper is deprecated and in yarp 3.8 it will be removed.

Reply:
Let's keep it then for the time being!
Thanks!

Reply:
>I'd suggest removing it since the RGBDSensorWrapper is deprecated and in yarp 3.8 it will be removed.

Sorry, I've missed your comment @mfussi66.
Ok, let's remove the file then!

Can you do that @davidetome?